src/View/Helper/SchemaHelper.php

```diff
@@ -165,6 +166,27 @@ public function controlOptions($name, $value, $schema) : array
                 'time' => 'true',
             ];
         } elseif ($type === 'checkbox') {
+            if (!empty($schema['oneOf'])) {
+                $options = [];
+                foreach ($schema['oneOf'] as $one) {
+                    if (!empty($one['type']) && ($one['type'] === 'array')) {
+                        $options = array_map(
+                            function ($value) {
+                                return ['value' => $value, 'text' => Inflector::humanize($value)];
+                            },
+                            $one['items']['enum']
```

Comment:
`$one['items']['enum']` may not exist
I'll add a check, something like
`Hash::extract($one, 'items.enum')`